src/globals/scss/core/_prose-scope.scss

```diff
@@ -0,0 +1,48 @@
+
+@include exports("prose-scope") {
+  .govuk-prose-scope {
+
+    // @extend inheritance
+
+    // All contextual heading and paragraph combinations are getting inherited
+    // when .govuk-body is getting extended
```

Comment:
The contextual adjustments are not just related to `.govuk-body` so I think this should just say something like

> Contextual heading and paragraph combinations are inherited through the use of @extend